modules/gdscript/gdscript_parser.cpp

```diff
@@ -4174,6 +4178,44 @@ bool GDScriptParser::warning_annotations(const AnnotationNode *p_annotation, Nod
 #endif // DEBUG_ENABLED
 }
 
+bool GDScriptParser::tool_button_annotation(const AnnotationNode *p_annotation, Node *p_node) {
+#ifndef TOOLS_ENABLED // !TOOLS_ENABLED
```

Comment:
I'd flip the condition here, with both cases having the negative condition *first* is confusing